app/src/main/res/layout/details_fragment.xml

```diff
@@ -0,0 +1,177 @@
+
+    <data class=".DetailBinding">
+
+        <variable
+            name="viewModel"
+            type="com.example.hungrywolfes.ui.detailsScreen.DetailsViewModel" />
+    </data>
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/grey"
+        android:fillViewport="true">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/constraint_layout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <FrameLayout
+                android:id="@+id/layout_heart"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:onClick="@{() -> viewModel.onFavoriteButton()}"
+                android:padding="@dimen/padding_large"
+                app:layout_constraintBottom_toTopOf="@+id/meal_image"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.97"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent">
+
+                <Button
+                    android:id="@+id/favorite_button"
+                    android:layout_width="@dimen/button_medium_width"
+                    android:layout_height="@dimen/button_medium_height"
+                    android:layout_marginEnd="@dimen/margin_small"
+                    android:background="@drawable/ic_favorite_heart"
+                    android:focusable="true"
+                    android:onClick="@{() -> viewModel.onFavoriteButton()}" />
+            </FrameLayout>
+
+            <FrameLayout
+                android:id="@+id/button_layout"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/margin_medium"
+                android:layout_marginTop="@dimen/margin_small"
+                android:onClick="@{()->viewModel.navigateBack()}"
+                android:padding="@dimen/padding_large"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.029"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent">
+
+                <Button
+                    android:id="@+id/back_button"
+                    android:layout_width="@dimen/back_button_small_width"
+                    android:layout_height="@dimen/back_button_small_height"
+                    android:background="@drawable/ic_backbutton"
+                    android:onClick="@{()->viewModel.navigateBack()}" />
+            </FrameLayout>
+
+            <de.hdodenhof.circleimageview.CircleImageView
+                android:id="@+id/meal_image"
+                android:layout_width="232dp"
```

Comment:
Indeed, it could work like this, but please extract the dimens.
Also there's a much better way. 
width and height set to 0dp & width_percent = X & dimensionRation=1:1
Please try this out and see how it works.